src/js/components/sharedComponents/header/InfoBanner.jsx

```diff
@@ -34,23 +35,11 @@ export default class InfoBanner extends React.Component {
                         <FontAwesomeIcon size="lg" icon="info-circle" />
                     </span>
                     <div className="info-banner__alert-text">
-                        <div className="info-banner__title-text">
-                            Coming to USAspending in July -- New Tools to Search and Display COVID-19 Spending Data:
-                        </div>
-                        Based on&nbsp;
-                        <a
-                            href="https://www.whitehouse.gov/wp-content/uploads/2020/04/Implementation-Guidance-for-Supplemental-Funding-Provided-in-Response.pdf"
-                            target="_blank"
-                            rel="noopener noreferrer"
-                            onClick={this.clickedBannerLink}>
-                            OMB Memo M-20-21
-                        </a>
-                        <span className="info-banner__description-external-link">
-                            <FontAwesomeIcon icon="external-link-alt" />
-                        </span>
-                        , Federal agencies will begin supplementing existing reporting of
-                        spending related to the Coronavirus Aid, Relief, and Economic Security (CARES) Act, as
-                        well as other COVID-19 appropriations in July 2020.
+                        <h2 className="info-banner__title-text">New to USAspending: Official COVID-19 Response Data</h2>
```

Comment:
Not sure this makes sense as an `h2` (since it's before the page `h1` in DOM). The margin is also adding more padding below the first line of text than in the previous banner.